doc/prop.xml

```diff
@@ -8,6 +8,83 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+    We will consider a digraph as a partial order with relation defined by
+    <A>x</A> <M>\leq</M> <A>y</A> if and only if <A>(x,y)</A> is an edge.
```

Comment:
In this package we use the notation `[x, y]` to define an edge. So overall, I think this would be better:

For a partial order digraph `<A>digraph</A>`, the corresponding partial order is the relation `<M>\leq</M>`, defined by `<M>x \leq y</M>` if and only if `<C>[x, y]</C>` is an edge of `<A>digraph</A>`.